modules/core/src/index.js

```diff
@@ -67,9 +67,8 @@ export {default as _OrthographicController} from './controllers/orthographic-con
 // EXPERIMENTAL EXPORTS
 
 // Experimental Effects (non-React) bindings
-export {default as _EffectManager} from './experimental/lib/effect-manager';
-export {default as _Effect} from './experimental/lib/effect';
-export {default as _ReflectionEffect} from './experimental/reflection-effect/reflection-effect';
+export {default as EffectManager} from './effects/effect-manager';
```

Comment:
Not sure how `EffectManager` will be useful to the user. Keep it experimental for now?

Reply:
I also don't see a need to export it. Assuming we don't export LayerManager, ViewManager etc.

Reply:
We do export `LayerManager` but documentation is lacking

Reply:
About that... We actually do have a [doc](https://github.com/uber/deck.gl/blob/master/docs/api-reference/internal/layer-manager.md) in our github doc strcuture but I suspect it is outdated and not published on the website.

I hesitate to just delete an already written doc, but we should probably move it into dev-docs, and remove the LayerManager export for the v7 release. 

@jianhuang01 	could you take care of that?

Reply:
ok, I will do it in the coming LayerManager clean up PR